src/cpu/x64/brgemm/jit_brgemm_amx_uker.cpp

```diff
@@ -1118,9 +1124,14 @@ void jit_brgemm_amx_uker_base_t::prefetch_CD_range(brgemm_iteration_t &bi,
             auto ptr_D = EVEX_compress_addr(reg_D, d_offset);
             uni_prefetch(ptr_D, pft, true);
         } else if (are_post_ops_applicable_) {
-            const auto c_offset = C_offset(bi, bdb, bd, ldb_pos);
-            auto ptr_C = EVEX_compress_addr(reg_C, c_offset);
-            uni_prefetch(ptr_C, pft, true);
+            //            todo: split hints C and D hints
```

Comment:
Please update other places, that's the way it's spelled and searched.
```suggestion
            //            TODO: split hints C and D hints
```

Reply:
fixed